apps/native/app/src/screens/document-detail/utils/share-file.tsx

```diff
@@ -0,0 +1,34 @@
+import Share from 'react-native-share'
+
+import { isAndroid } from '../../../utils/devices'
+import { authStore } from '../../../stores/auth-store'
+import { DocumentV2 } from '../../../graphql/types/schema'
+
+interface ShareFileProps {
+  document: DocumentV2
+  hasPdf: boolean
```

Comment:
this should be unnecessary you could just have pdfUrl as optional and update the logic within the method